qa/suites/rbd/ro/workloads/immutable_object_cache.yaml

```diff
@@ -0,0 +1,24 @@
+tasks:
+- install:
```

Comment:
Nit: pull all the common config / test setup to a single yaml

Reply:
merge qemu test and fio test to a single yaml?

Reply:
> Nit: pull all the common config / test setup to a single yaml

merge qemu test and fio test to a single yaml?

Reply:
No, avoid duplicating these bootstrapping actions by pulling the common bits out of the two workload YAMLs and into its own YAML file in its own sub-directory.